pandas/tests/series/methods/test_replace.py

```diff
@@ -16,7 +16,7 @@ def test_replace_explicit_none(self):
         expected = pd.Series([0, 0, None], dtype=object)
         tm.assert_series_equal(result, expected)
 
-        df = pd.DataFrame(np.zeros((3, 3)))
+        df = pd.DataFrame(np.zeros((3, 3))).astype({2: object})
         df.iloc[2, 2] = ""
```

Comment:
the third column `df[2]` is initially made up of ints, and then an element is set to  `' '` (thus upcasting to object)

might as well set to `object` from the start, as the purpose of the test is on the line below `df.replace(...`